webpack.config.js

```diff
@@ -22,36 +22,46 @@ module.exports = {
   // },
   module: {
     rules: [
+      // {
+      //   test: /\.jsx?$/,
+      //   exclude: ['node_modules'],
+      //   use: ['babel-loader'],
+      // },
       {
-        test: /\.jsx?$/,
-        exclude: ['node_modules'],
-        use: ['babel-loader'],
-      },
-      {
-        test: /\.css$/,
+        test: /\.css$/i,
         use: ['style-loader', 'css-loader'],
       },
       {
-        test: /\.gif$/,
-        type: 'asset/inline',
+        test: /\.png/,
+        type: 'asset/resource',
       },
       {
-        test: /\.(ttf|eot|svg)$/,
-        type: 'asset/resource',
+        test: /\.html$/,
+        use: [
+          'html-loader',
+        ],
       },
+      // {
+      //   test: /\.gif$/,
+      //   type: 'asset/inline',
+      // },
+      // {
+      //   test: /\.(ttf|eot|svg)$/,
+      //   type: 'asset/resource',
+      // },
     ],
   },
-  resolve: {
-    alias: {
-      config$: './configs/app-config.js',
-      react: './vendor/react-master',
-    },
-    extensions: ['.js', '.jsx'],
-    modules: [
-      'node_modules',
-      'bower_components',
-      'shared',
-      '/shared/vendor/modules',
-    ],
-  },
+  // resolve: {
+  //   alias: {
+  //     config$: './configs/app-config.js',
+  //     react: './vendor/react-master',
+  //   },
+  //   extensions: ['.js', '.jsx'],
+  //   modules: [
+  //     'node_modules',
+  //     'bower_components',
+  //     'shared',
+  //     '/shared/vendor/modules',
+  //   ],
+  // },
```

Comment:
I suggest you remove all commented code in your files. Removing them will help improve code readability.

Reply:
Some are removed some are used. 😉